cobbler/utils.py

```diff
@@ -221,6 +221,32 @@ def is_systemd() -> bool:
     return False
 
 
+def is_supervisord() -> bool:
+    """
+    Return whether or not this system uses systemd.
+
+    This method currently checks if the path ``/usr/lib/systemd/systemd`` exists.
+    """
+    if os.path.exists("/usr/bin/supervisord"):
+        return True
+    return False
+
+
+def service_restart(service_name):
+
+    if is_systemd():
+        restart_command = "systemctl restart " + service_name
+    # elif is_supervisord():
+    # FIXME what needs to be done for supervisord?
+    else:
+        restart_command = "service " + service_name + " restart"
```

Comment:
This will fail if service is not available. So the `else` branch would be a `logger.warning` that the service was not restarted and then the method returns imho.